app/src/lib/git/add.ts

```diff
@@ -12,5 +12,5 @@ export async function addConflictedFile(
   repository: Repository,
   file: WorkingDirectoryFileChange
 ) {
-  await git(['add', '--', file.path], repository.path, 'addConflictedFile')
+  await git(['add', '--', file.path], repository.path, 'addConflictedFile', {}, repository.codespace)
```

Comment:
I use the empty object a _lot_ to hold these options...and I forgot about the empty object being truthy. This should almost certainly be `undefined` everywhere.